### ISSUE TITLE
update to cuda 12.4 ? 

### ISSUE THREAD

**Issue by stevanisya** [muzishen/IMAGDressing#55]
Hello, I'm just wondering if you would update from cuda 11.8 to 12.4 ? 
because my computer is installed with 12.4 and sadly I can't have 2 different cuda versions in 1 computer. :/ 
Thanks !

**Comment by nitinmukesh** [comment]
Yes you can have 2 CUDA versions in same computer. I am using multiple versions on Windows. 11.8, 12.1, 12.4..

But yes 12.4 is good to have as it reduces the models memory requirement.
